apps/designer/app/designer/features/sidebar-left/panels/pages/pages.tsx

```diff
@@ -129,14 +164,19 @@ export const TabContent = (props: TabContentProps) => {
   const [project] = useProject();
 
   const navigate = useNavigate();
-  const handleSelect = (pageId: string) => {
+  const handleSelect = (pageId: "home" | Page["id"]) => {
     if (project === undefined) {
       return;
     }
-    navigate(`${props.config.designerPath}/${project.id}?pageId=${pageId}`);
+    if (pageId === "home") {
+      navigate(`${props.config.designerPath}/${project.id}`);
+    } else {
+      navigate(`${props.config.designerPath}/${project.id}?pageId=${pageId}`);
+    }
   };
 
-  const [newPageOpen, setNewPageOpen] = useState(false);
+  const NEW_PAGE = "new-page";
```

Comment:
we usually use this naming convention only when we export those constants to be reused across project, so that they are immediately visible as contacts, inline - just normal camel case